src/components/Todos/VisibleTodos.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { filterTodo } from '../../utils/filterTodos';
+
+type Props = {
+  todos: Todo[],
+  onRemove: (todoId: number) => void,
+  onComplete: (todoId: number, title: string, completed: boolean) => void,
+  onTodoChange: (event: React.ChangeEvent<HTMLInputElement>,
+    todoId: number) => void,
+  onCurrentTodoChange: (event: React.FormEvent<HTMLFormElement> | null,
+    title: string, todoId: number) => void,
+  isForm: boolean,
+  onFormComplete: () => void,
+  status: string,
+  activeTodoId: number[],
+};
+
+export const VisibleTodos: React.FC<Props> = ({
```

Comment:
is this component used anywhere in the project?

Reply:
Deleted...forgot to delete from previos parts